parking/backend/engine/alloc_engine.py

```diff
@@ -0,0 +1,58 @@
+import logging
+from typing import Optional
+from geopy.distance import distance
+from parking.backend.db.dbaccess import DbAccess
+from parking.shared.rest_models import ParkingLot
+from parking.shared.location import Location
+from parking.shared.ws_models import ParkingRequestMessage
+
+logger = logging.getLogger('backend')
+
+
+class AllocationEngine():
```

Comment:
Should the engine handle all connections to the DbAccess? 

i.e. anything going through the REST API should go through the `AllocationEngine` ?

Reply:
The engine is only going to be used for allocations, so I think it's fine for the REST API to communicate with the DB directly.